practice-app/client/src/components/Meric/ForecastSearch.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import { fetchLocationInfo, fetchForecastInfo, saveForecast } from "../../queries/meric.query";
+import TextField from "@mui/material/TextField";
+import { Button, MenuItem } from "@mui/material";
+
+function ForecastSearch() {
+
+    const [searchText, setSearchText] = useState("")
+    const [options, setOptions] = useState([]);
+    const [selectedOption, setSelectedOption]= useState({});
+
+    const handleInputChange = async (e) => {
+
+        console.log(e.target.value);
+        const newInput = e.target.value;
+        setSearchText(newInput);
+        const locationInfo = await fetchLocationInfo((newInput));
+        setOptions(locationInfo);
+
+    }
+
+    const handleSelect = async (e) => {
+
+        setSelectedOption(options.find((option) => option.key === e.target.value));
+
+    }
+
+    const handleSubmit = async (e) => {
+
+        e.preventDefault();
+        const forecastInfo = await fetchForecastInfo((selectedOption.key));
+        const status = await saveForecast(selectedOption.city, selectedOption.country, forecastInfo[0].high, forecastInfo[0].low, new Date());
+
+    }
+
+    return (
+        <form
+            style={{ display: 'flex', flexDirection: 'column', justifyContent: 'space-evenly' }}
```

Comment:
instead of giving style like this, we may use bootstrap classes such as d-flex, justify-content-center and etc.. otherwise it keeps hard to watch styling.